### PR TITLE
Use pointer to support empty value

### PR DESCRIPTION
**Pull request by baijum**
Signed-off-by: Baiju Muthukadan <baiju.m.mail@gmail.com>

### CODE REVIEW

File: apis/v1alpha3/zz_generated.deepcopy.go [hunk 1]
```diff
@@ -1,4 +1,3 @@
-//go:build !ignore_autogenerated
```
Comment (scothis) [inline apis/v1alpha3/zz_generated.deepcopy.go:1]: either restore this line manually, or install go 1.17 on your machine.

**Comment by baijum** [comment]
Not sure why the build is failing. I am still using Go 1.16 locally. I will upgrade to 1.17 and check again.

File: apis/v1alpha3/zz_generated.deepcopy.go (hunk 1 above)
Comment (baijum) [reply apis/v1alpha3/zz_generated.deepcopy.go:1]: ok,  I will upgrade to 1.17 and check.

Comment (baijum) [reply apis/v1alpha3/zz_generated.deepcopy.go]: Upgrading to Go 1.17 fixed the issue. Thanks!

**Comment by baijum** [comment]
> Not sure why the build is failing. I am still using Go 1.16 locally. I will upgrade to 1.17 and check again.

Upgrading to Go 1.17 fixed the issue.

**Pull request merged**